### PR TITLE
Fix/footer links

### LINKED ISSUE #734: Fix links to community
@sseerrggii it seems that https://community.coopdevs.org is down 🤔? In that case we should think something to replace a couple of mentions: https://github.com/search?q=repo%3Acoopdevs%2Ftimeoverflow%20community.coopdevs.org&type=code, assuming we don't want the Discourse back.

### PR DESCRIPTION
**Pull request by sseerrggii**
Some links update 

Closes #734

### CODE REVIEW

**Comment by sseerrggii** [comment]
@markets I'm not sure why I update also db/structure.sql

it wasn't my intention :sweat_smile:

File: app/views/application/_footer.html.erb [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
       <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4 footer-left-col">
-        <a href="http://community.coopdevs.org/" target="_blank"><%= t('layouts.application.help') %></a>
+        <a href="https://adbdt.org/contacta/" target="_blank"><%= t('layouts.application.help') %></a>
         <%= link_to t('application.terms_conditions'), tnc_path, data: { target: '#modal', toggle: 'modal' } %>
       </div>
       <div class="col-xs-12 col-sm-2 col-md-2 col-lg-4 footer-center-col">
         <%= render 'application/menus/language_switcher' %>
       </div>
       <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4 footer-right-col">
-        <a href="https://twitter.com/timeoverflow" target="_blank">Twitter</a>
```
Comment (markets) [inline app/views/application/_footer.html.erb:12]: no more Twitter account?

File: db/structure.sql [hunk 2]
```diff
@@ -1378,3 +1373,5 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20230401114456'),
 ('20231120164231'),
 ('20231120164346');
+
+
```
Comment (markets) [inline db/structure.sql:1377]: Yeah, not sure why this happens, probably due to the different PostgreSQL versions we used during the migration to the new infra... Maybe we can revert this for now? I think we should solve it separately.

File: app/views/application/_footer.html.erb (hunk 1 above)
Comment (sseerrggii) [reply app/views/application/_footer.html.erb:12]: We don't use it, so I prefer not to have this link

File: db/structure.sql (hunk 2 above)
Comment (sseerrggii) [reply db/structure.sql:1377]: Yes, could you do it?

Comment (markets) [reply db/structure.sql:1377]: Could you please 🙏🏼 restore it by coping the file from develop branch?

Comment (sseerrggii) [reply db/structure.sql]: Done

**Pull request merged**